hack/verify/validate-ocp-version.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+import re
+import sys
+import yaml
+
+PLUGIN_CONFIG = 'pluginconfig/pluginconfig-311.yaml'
+
+
+def main():
+    config = yaml.load(open(PLUGIN_CONFIG, 'r'))
```

Comment:
naughty:
```
hack/verify/validate-ocp-version.py:11: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  config = yaml.load(open(PLUGIN_CONFIG, 'r'))
```

Reply:
I think I have a very old version of pyyaml locally :sweat_smile: